application/forms/PreferenceForm.php

```diff
@@ -208,8 +208,15 @@ public function createElements(array $formData)
                     )
                 );
             }
+        } else {
+            $globalThemeFile = Config::app()->get('themes', 'default');
+            $themeFile = StyleSheet::getThemeFile($globalThemeFile);
+            $file = $themeFile !== null ? @file_get_contents($themeFile) : false;
+            if ($file && strpos($file, StyleSheet::LIGHT_MODE_IDENTIFIER) === false) {
+                $disabled = ['', 'light', 'system'];
+            }
         }
-
+        
```

Comment:
Please don't add superfluous whitespace.

Reply:
I mean the indentation, not the empty line itself, if you've wondered.

Reply:
I can still see the indentation: :thinking: 
![Screenshot from 2022-03-29 12-45-06](https://user-images.githubusercontent.com/16668527/160594843-4b95ae30-752e-4236-9ae7-4f185063cad1.png)